test/libmpv_common.h

```diff
@@ -65,57 +65,94 @@ static inline mpv_event *wrap_wait_event(void)
     }
 }
 
-static inline void check_api_error(int status)
+static inline void command(const char *cmd[])
 {
-    if (status < 0)
-        fail("mpv API error: %s\n", mpv_error_string(status));
+    int ret = mpv_command(ctx, cmd);
+    if (ret < 0)
+        fail("mpv API error while running command '%s': %s\n", cmd[0], mpv_error_string(ret));
```

Comment:
Can you also call `fflush` in `fail()`? meson likes to buffer data and it's lost later.